GameData/RP-0/CustomBarnKit.cfg

```diff
@@ -90,11 +90,11 @@
 	}
 	@ADMINISTRATION
 	{
-		@levels = 2
-		@upgradesVisual = 3, 3
-		@upgrades = 38000, 38000
-		@activeStrategyLimit = 1, 1
-		@strategyCommitRange = 0, 0
+		@levels = 3
+		@upgradesVisual = 1, 2, 3
+		@upgrades = 1, 1 // purely visual, so basically free
```

Comment:
Pretty sure this should have 3 values. The cost of 1st level is still used for determining facility repairs.